packages/rest/src/rest-server.ts

```diff
@@ -131,20 +131,24 @@ export class RestServer extends Context implements Server {
   ) {
     super(app);
 
-    if (!options) options = {};
+    options = options || {};
 
     // Can't check falsiness, 0 is a valid port.
-    if (options.port === null || options.port === undefined) {
+    if (options.port == null) {
       options.port = 3000;
     }
+    if (options.host == null) {
+      options.host = 'localhost';
+    }
     this.bind(RestBindings.PORT).to(options.port);
+    this.bind(RestBindings.HOST).to(options.host);
     this.api(createEmptyApiSpec());
 
     this.sequence(options.sequence ? options.sequence : DefaultSequence);
 
     this.handleHttp = (req: ServerRequest, res: ServerResponse) => {
       try {
-        this._handleHttpRequest(req, res).catch(err =>
+        this._handleHttpRequest(req, res, options!).catch(err =>
```

Comment:
What is the `!` for at the end of `options`?

Reply:
It tells TypeScript `options` is not undefined.